pkgs/applications/science/misc/megam/default.nix

```diff
@@ -22,33 +26,39 @@ stdenv.mkDerivation {
   '';
   strictDeps = true;
 
-  nativeBuildInputs = [ makeWrapper ocaml ];
+  nativeBuildInputs = [ ocaml ];
 
   buildInputs = [ ncurses ];
 
   makeFlags = [
     "CAML_INCLUDES=${ocaml}/lib/ocaml/caml"
     ("WITHBIGARRAY=" + lib.optionalString (lib.versionOlder ocaml.version "4.08.0") "bigarray.cma")
+    "all"
+    "opt"
   ];
 
   # see https://bugzilla.redhat.com/show_bug.cgi?id=435559
   dontStrip = true;
 
   installPhase = ''
-    mkdir -pv $out/bin
-    cp -Rv megam $out/bin
+    runHook preInstall
+
+    install -Dm755 megam $out/bin/megam
+    install -Dm755 megam.opt $out/bin/megam.opt
+
+    runHook postInstall
   '';
 
 
-  meta = {
+  meta = with lib; {
     description = "MEGA Model Optimization Package";
 
-    longDescription =
-      ''  The software here is an implementation of maximum likelihood
-          and maximum a posterior optimization of the parameters of
-          these models.  The algorithms used are much more efficient
-          than the iterative scaling techniques used in almost every
-          other maxent package out there.  '';
+    longDescription = ''
+      The software here is an implementation of maximum likelihood and maximum a
+      posterior optimization of the parameters of these models. The algorithms
+      used are much more efficient than the iterative scaling techniques used in
+      almost every other maxent package out there.
+    '';
 
```

Comment:
We could trim the empty lines here and do you want to add yourself as a maintainer?